salt/metalk8s/volumes/prepared/init.sls

```diff
@@ -0,0 +1,51 @@
+include:
+  - .installed
+
+Refresh pillars for all minions:
+  salt.function:
+    - name: saltutil.refresh_pillar
+    - tgt: '*'
```

Comment:
Why?

Reply:
We wanted to be sure that we had an up-to-date pillar at that point, but I guess it's useless/won't do what we want?

Reply:
There should be no need. The pillar should be recalculated when `state.sls` is invoked. Also, the `tgt: '*'` is quite unexpected. Not even sure how this can work in a `state.sls`-interpreted SLS.

Reply:
Ok, will remove it then.